DigitalLearningSolutions.Web/Views/Shared/PaginatedPage/_BottomPaginationNavWithoutAlert.cshtml

```diff
@@ -0,0 +1,6 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.Common.SearchablePage
+@model IBasePaginatedViewModel
+
+<nav class="nhsuk-pagination @(Model.TotalPages == 1 ? "pagination-hidden" : "")" role="navigation" aria-label="Bottom Pagination">

```

Comment:
Is the aria-label just there to determine what the screen-reader reads out?

Reply:
The motivation behind my question is, "It would be nice if we can simplify the codebase by making the top and bottom ones the same.

Reply:
Also so that the landmarks are unique. We get accessibility errors when both navigation have the same label as they are not unique.
"Landmarks must have a unique role or role/label/title (i.e. accessible name) combination"